### PR TITLE
Add Osmosis Testnet

### PR DESCRIPTION
**Pull request by JeremyParish69**

### CODE REVIEW

**Comment by sunnya97** [comment]
@czarcas7ic does this look right to you?

File: testnets/osmosistestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                "denom": "uosmo"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/osmosis-labs/osmosis",
+        "recommended_version": "v7.0.2",
+        "compatible_versions": [
+            "v7.0.2"
+        ],
```
Comment (sunnya97) [inline testnets/osmosistestnet/chain.json:41]: I don't think this is right.  I'm pretty sure osmosis testnet is on v11

Comment (czarcas7ic) [reply testnets/osmosistestnet/chain.json:41]: Correct, osmo-test-4 is on v11 currently

**Comment by pyramation** [comment]
small nit pick — should we standardize on something like `<chainname>-testnet`, in this case would be `osmosis-testnet` with hyphen? 

just my 2c, happy either way ;)

**Comment by JeremyParish69** [comment]
> small nit pick — should we standardize on something like `<chainname>-testnet`, in this case would be `osmosis-testnet` with hyphen?
> 
> just my 2c, happy either way ;)
> 
> UPDATE: now I'm not sure if hyphens are even allowed or exist in other chain names.... potentially counters my nit pick.

At some point I heard that chain name would be restricted to alphanumeric (no hyphens)

**Comment by sunnya97** [comment]
@pyramation yeah I made a rule that no hyphens in chainnames, to make it not a bikeshed of whether it should be like `cosmoshub` or `cosmos-hub` lol

**Comment by pyramation** [comment]
> @pyramation yeah I made a rule that no hyphens in chainnames, to make it not a bikeshed of whether it should be like `cosmoshub` or `cosmos-hub` lol

cool that works!

**Pull request merged**